zookeeper-server/src/main/java/org/apache/zookeeper/server/ServerMetrics.java

```diff
@@ -444,6 +449,14 @@ private ServerMetrics(MetricsProvider metricsProvider) {
     public final Counter REQUEST_THROTTLE_WAIT_COUNT;
     public final Counter LARGE_REQUESTS_REJECTED;
 
+    /*
+     * Client Auth requests for x509 based AuthenticationProviders through portUnification.
+     */
+    public final Counter UNIFIED_PORT_NONSSL_REQUESTS;
```

Comment:
just curious, don't you have to initialize it to 0 (or that's default)

Reply:
Yup it's by-default set to 0.